src/EnergyPlus/WaterToAirHeatPump.cc

```diff
@@ -208,24 +208,13 @@ namespace WaterToAirHeatPump {
         int HPNum; // The Water to Air HP that you are currently loading input into
         int NumCool;
         int NumHeat;
```

Comment:
I wish I would have told you to hold off on this until I was done with what I was doing. Too late. Get this merged and I will deal with it. Lesson learned.